src/cmd/destroy.go

```diff
@@ -57,7 +57,7 @@ var destroyCmd = &cobra.Command{
 			// Iterate over all matching zarf-clean scripts and exec them
 			for _, script := range scripts {
 				// Run the matched script
-				_, _, err := utils.ExecCommandWithContext(context.TODO(), true, script)
+				err := exec.CmdWithPrint(script)
```

Comment:
So I actually tried to do this, but it gets super messy because you essentially need the original init package/scripts to do a proper `zarf destroy`. Keeping the scripts this way is actually more reliable for doing a final clean sweep. Also, since this function currently doesn't run package remove actions properly, I think it's out of scope to solve that problem in this PR too.